v2/internal/attacktechniques/gcp/exfiltration/share-compute-image/main.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 4.28.0"
+    }
+  }
+}
+
+locals {
+  image-name = "stratus-red-team-victim-image"
+}
+
+resource "google_compute_image" "this" {
+  name = local.image-name
+
+  raw_disk {
+    source = "https://storage.googleapis.com/bosh-gce-raw-stemcells/bosh-stemcell-97.98-google-kvm-ubuntu-xenial-go_agent-raw-1557960142.tar.gz"
```

Comment:
where is this coming from?

Reply:
I copied it from https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_image 
This is something I still need to look into in order to make it faster

Reply:
I don't have a better way, so I'd suggest we merge as it - good for you?